README.md

```diff
@@ -1,7 +1,8 @@
 # PrairieLearn
 
-PrairieLearn is an online problem-driven learning system for creating homeworks and tests. It allows questions to be written using arbitrary HTML/JavaScript, thus enabling very powerful questions that can randomize and autograde themselves, and can access client- and server-side libraries to handle tasks such as graphical drawing, symbolic algebra, and student code compilation and execution.
+PrairieLearn is an online problem-driven learning system for creating homeworks and tests. It allows questions to be written using arbitrary HTML, JavaScript, and Python, thus enabling very powerful questions that can randomize and autograde themselves, and can access client- and server-side libraries to handle tasks such as graphical drawing, symbolic algebra, and student code compilation and execution.
 
-[![Latest version](https://img.shields.io/github/tag/PrairieLearn/PrairieLearn.svg?label=version)](https://github.com/PrairieLearn/PrairieLearn/blob/master/ChangeLog.md) [![Dependabot Status](https://api.dependabot.com/badges/status?host=github&repo=PrairieLearn/PrairieLearn)](https://dependabot.com) [![Docker build status](https://img.shields.io/docker/automated/prairielearn/prairielearn.svg)](https://hub.docker.com/r/prairielearn/prairielearn/builds/) [![Build Status](https://img.shields.io/travis/PrairieLearn/PrairieLearn/master.svg)](https://travis-ci.org/PrairieLearn/PrairieLearn) [![Coverage Status](https://coveralls.io/repos/github/PrairieLearn/PrairieLearn/badge.svg?branch=master)](https://coveralls.io/github/PrairieLearn/PrairieLearn?branch=master) [![License](https://img.shields.io/github/license/PrairieLearn/PrairieLearn.svg)](https://github.com/PrairieLearn/PrairieLearn/blob/master/LICENSE)
+[![GitHub Actions build status](https://github.com/PrairieLearn/PrairieLearn/actions/workflows/main.yml/badge.svg)](https://github.com/PrairieLearn/PrairieLearn/actions/workflows/main.yml) [![License](https://img.shields.io/github/license/PrairieLearn/PrairieLearn.svg)](https://github.com/PrairieLearn/PrairieLearn/blob/master/LICENSE)
```

Comment:
I'm worried about this change because it's repeating a change made by @nwalters512 in #6002. I suspect there might be something wrong with the git branch you are working on?